client/src/app/(onboarding)/signup/page.tsx

```diff
@@ -1,3 +1,211 @@
+          sx={{
+            width: "100px",
+            height: "1px",
+            backgroundColor: "#ccc",
+            marginLeft: 2,
+          }}
+        />
+      </div>
+
+      {/* Signup form */}
+      <div className="signup-form">
+        {/* Email field */}
+        <div className="text-field">
+          <Typography
+            variant="subtitle1"
+            sx={{ fontWeight: "bold", marginBottom: "8px" }}
+          >
+            Email
+          </Typography>
+          <TextField
+            fullWidth
+            label="Email"
+            placeholder="Enter your email"
+            required
+            error={!!error.email} // Check if there is an error
+            helperText={error.email} // Display error message
+            value={email}
+            onChange={(e) => setEmail(e.target.value)} // Update email state
+          />
+        </div>
+
+        {/* Password field */}
+        <div className="text-field">
+          <Typography
+            variant="subtitle1"
+            sx={{ fontWeight: "bold", marginBottom: "8px" }}
+          >
+            Password 
+          </Typography>
+          <TextField
+            fullWidth
+            label="Password"
+            type="password" // Ensure this is password input
+            placeholder="Enter your password"
+            required
+            error={!!error.password} // Check if there is an error
+            helperText={error.password} // Display error message
+            value={password}
+            onChange={(e) => setPassword(e.target.value)} // Update password state
+          />
+        </div>
+
+        {/* Confirm Password field */}
+        <div className="text-field">
+          <Typography
+            variant="subtitle1"
+            sx={{ fontWeight: "bold", marginBottom: "8px" }}
+          >
+            Confirm Password
+          </Typography>
+          <TextField
+            fullWidth
+            label="Confirm Password"
+            type="password" // Ensure this is password input
+            placeholder="Confirm your password"
+            required
+            error={!!error.confirmPassword} // Check if there is an error
+            helperText={error.confirmPassword} // Display error message
+            value={confirmPassword}
+            onChange={(e) => setConfirmPassword(e.target.value)} // Update confirm password state
+          />
+        </div>
+
+        {/* Submit button */}
+        <Button
+          className="borderline"
+          type="button"
+          variant="contained"
+          color="primary"
+          onClick={handleSubmit} // Trigger submit logic on click
+          sx={{
+            mt: 2,
+            width: "200px",
+            boxShadow: "none",
+            backgroundColor: "#496FFF",
+            ":hover": {
+              backgroundColor: "#3B5AC6",
+              boxShadow: "none",
+            },
+          }}
+        >
+          Confirm
+        </Button>
+
+        <Typography
```

Comment:
Almost missed this. The Link should be the wrapper. Not the Typography

Reply:
Yeah it has been fixed!